v3/src/components/graph/components/graph.tsx

```diff
@@ -121,12 +122,16 @@ export const Graph = observer((
   }
 
   function hideDataTip(event: MouseEvent) {
-    const [, caseID] = select(event.target as SVGSVGElement).property('id').split("_"),
-      isSelected = dataset?.isCaseSelected(caseID)
+    const target = select(event.target as SVGSVGElement)
     dataTip.hide()
-    select(event.target as SVGSVGElement)
-      .transition().duration(transitionDuration)
-      .attr('r', isSelected ? selectedPointRadius : pointRadius)
+    if (target.node()?.nodeName === 'circle' && dataset && !active(target.node()) &&
+      !target.property('isDragging')) {
```

Comment:
Since this same test occurs twice (and as this PR demonstrates, occasionally they have to be modified in tandem), it seems like a utility function might be useful.

Reply:
Done!